sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_schema_registry_avro_serializer.py

```diff
@@ -120,29 +120,30 @@ def _get_schema(self, schema_id, **kwargs):
             self._schema_to_id[schema_str] = schema_id
             return schema_str
 
-    def serialize(self, data, schema, **kwargs):
+    def serialize(self, value, schema, **kwargs):
         # type: (Dict[str, Any], Union[str, bytes], Any) -> bytes
         """
-        Encode dict data with the given schema. The returns bytes are consisted of: The first 4 bytes
+        Encode data with the given schema. The returns bytes are consisted of: The first 4 bytes
         denoting record format identifier. The following 32 bytes denoting schema id returned by schema registry
         service. The remaining bytes are the real data payload.
 
-        :param data: The dict data to be encoded.
+        :param value: The data to be encoded.
+        :type value: Dict[str, Any]
```

Comment:
as per OOB discussion, we need to investigate the input of Any type (python primitive types)